src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -60,7 +60,7 @@ namespace winrt::TerminalApp::implementation
         std::optional<int> _rearrangeFrom;
         std::optional<int> _rearrangeTo;
 
-        ShortcutActionDispatch _actionDispatch{};
+        winrt::com_ptr<ShortcutActionDispatch> _actionDispatch{ winrt::make_self<ShortcutActionDispatch>() };
```

Comment:
this perturbs me a bit. something about the complex initializer statement in the header, idk